file_paths.md

```diff
@@ -106,16 +115,144 @@ puppet-agent
+        facts.d                               # external facts directory (not pluginsync'ed)
+
+    C:\ProgramData\PuppetLabs\mcollective
+        plugins                               # user installed plugins
+
+    C:\ProgramData\PuppetLabs\puppet\var\log  # :logdir                      same
+        puppet.log                            # not enabled by default
+
+    C:\ProgramData\PuppetLabs\puppet\var\run  # :rundir                      same
+        agent.pid                             # :pidfile
+
+    C:\Program Files\Puppet Labs\Puppet\bin
+        cfacter.bat                           # bat file wrappers
+        facter.bat
+        hiera.bat
+        mco.bat
+        puppet.bat
+        environment.bat                       # setup LOAD_PATH
+        *_interactive.bat                     # targets for shortcuts
+
+    C:\Program Files\Puppet Labs\Puppet\cfacter
+        bin                                   # executables and dlls
+            cfacter.exe
+            libfacter.so
+            lib*.dll
+        inc                                   # cfacter headers
+            facter
+        lib
+            cfacter.rb                        # ruby bindings
+
+    C:\Program Files\Puppet Labs\Puppet\facter
+        bin
+            facter                            # ruby bin wrapper
+        lib
+            facter.rb
+
+    C:\Program Files\Puppet Labs\Puppet\hiera
+        bin
+            hiera                             # ruby bin wrapper
+        lib
+            hiera.rb
+
+    C:\Program Files\Puppet Labs\Puppet\mcollective
+        bin
+            mcollectived                      # ruby bin wrapper
+        lib
+            mcollective.rb
+
+    C:\Program Files\Puppet Labs\Puppet\misc
+        LICENSE.rtf                           # license
+        puppetlabs.ico                        # icon for start menu shortcut
+        puppetres.dll                         # event log message resource dll
+        versions.txt                          # versions of components
+
+    C:\Program Files\Puppet Labs\Puppet\puppet
+        bin
+            puppet                            # ruby bin wrapper
+        lib
+            puppet.rb
+
+    C:\Program Files\Puppet Labs\Puppet\service
+        daemon.rb                             # windows service daemon
+
+    C:\Program Files\Puppet Labs\Puppet\sys
+        ruby
+            bin
+                ruby.exe
+                ssleay32.dll                  # openssl dll
+            include
+            lib
+            share
+
+        tools
+            bin
+                elevate.exe                   # Used to elevate interactive commands
+
+
 These sections describe other Puppet packages that rely on puppet-agent to create the initial directory layout. It does not attempt to specify the full set of file paths for these packages, just cases where the other package has a dependency on puppet-agent.
 
-puppetdb
+<a id="puppet-db">puppet-db</a>
```

Comment:
You just have to remove punctuation and replace spaces with dashes...

https://github.com/Iristyle/Readme-test/blob/master/README.md#puppet-agent-windows